src/pages/messaging/agent-chat-api/rtm-reference/index.mdx

```diff
@@ -2439,12 +2523,22 @@ It returns the initial state of the current Agent.
 
 #### Specifics
 
-|                        |                                                                                        |
-| ---------------------- | -------------------------------------------------------------------------------------- |
-| **Action**             | `login`                                                                                |
-| **Required scopes**    | `chats--access:ro` `customers:ro` `multicast:ro` `agents--all:ro` `agents-bot--all:ro` |
-| **Web API equivalent** | -                                                                                      |
-| **Push message**       | -                                                                                      |
+|                        |                                                                                                                                                              |
+| ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **Action**             | `login`                                                                                                                                                      |
+| **Required scopes**    | `chats--access:ro` **\*** and `customers:ro` **\*\*** and `multicast:ro` **\*\*\*** and `agents--all:ro`**\*\*\*\*** and `agents-bot--all:ro` **\*\*\*\*\*** |
+| **Web API equivalent** | -                                                                                                                                                            |
+| **Push message**       | -                                                                                                                                                            |
+
+**\*)** `chats--access:ro` - to get pushes about chats from the requester groups.
+
+**\*\*)** `customers:ro` - to get pushes about customers.
+
+**\*\*\*)** `multicast:ro` - to get pushes about multicast.
+
+**\*\*\*\*)** `agents--all:ro` - to get pushes about agents updates.
```

Comment:
@bkubiak I have added 'about' in this description as without, it seemed weird. Is that okay? :)

Reply:
of course, it is ok - it was my typo in the doc in conflu and please don't pay attention to the doc in conflu when it comes to the style and language - its purpose is just a source of knowledge for you :)